platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/annotations/Annotation.java

```diff
@@ -70,18 +70,18 @@ public void setId(long id) {
   /**
    * Do not use this method, used internally by the SDK.
    *
-   * @param maplibreMap the hosting mapbox map
+   * @param maplibreMap the hosting MapLibreMap
    */
-  public void setMapboxMap(MapLibreMap maplibreMap) {
```

Comment:
I mean... The comment is pretty clear right.

Maybe we want to deprecate this first though.

Reply:
The whole class is deprecated so I'm not sure what use deprecating its individual methods is :)